Assets/MixedRealityToolkit/_Core/Devices/VoiceInput/WindowsSpeechInputDeviceManager.cs

```diff
@@ -116,10 +117,15 @@ public void StopRecognition()
 
 #if UNITY_STANDALONE_WIN || UNITY_WSA || UNITY_EDITOR_WIN
         private void KeywordRecognizer_OnPhraseRecognized(PhraseRecognizedEventArgs args) {
-            OnPhraseRecognized((RecognitionConfidenceLevel) args.confidence, args.phraseDuration, args.phraseStartTime, args.semanticMeanings, args.text);
+            List<KeyValuePair<string, string[]>> semanticMeanings = new List<KeyValuePair<string, string[]>>(args.semanticMeanings.Length);
```

Comment:
Not sure I like this, we're allocating new list every time we get a phrase recognized. This will impact the gc and the perf of this feature.

Reply:
Maybe we can remove completely for now?